plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2724,3 +2724,11 @@ function wc_update_891_create_plugin_autoinstall_history_option() {
 		}
 	}
 }
+
+/**
+ * Add woocommerce_show_lys_tour.
+ */
+function wc_update_900_add_launch_your_store_tour_option() {
+	update_option( 'woocommerce_show_lys_tour', 'yes' );
```

Comment:
What do you think of using [`add_option`](https://developer.wordpress.org/reference/functions/add_option/) instead? This can help to make it safer by setting the option only when it doesn't exist

Reply:
@ilyasfoo Good idea! Updated in [Use add_option](https://github.com/woocommerce/woocommerce/pull/47634/commits/9413825f5fbbfd4acee169775176c91c5f159675)